echopype/preprocess/api.py

```diff
@@ -156,29 +170,32 @@ def compute_MVBS_index_binning(ds_Sv, range_sample_num=100, ping_num=100):
         .mean(skipna=True)
     )
 
-    # Attach coarsened echo_range
+    # Attach attributes and coarsened echo_range
     da.name = "Sv"
-    ds_out = da.to_dataset()
-    ds_out["echo_range"] = (
+    ds_MVBS = da.to_dataset()
+    ds_MVBS.coords["range_sample"] = (
+        "range_sample",
+        np.arange(ds_MVBS["range_sample"].size),
+        {"long_name": "Along-range sample number, base 0"},
+    )  # reset range_sample to start from 0
+    ds_MVBS["echo_range"] = (
         ds_Sv["echo_range"]
         .coarsen(  # binned echo_range (use first value in each average bin)
             ping_time=ping_num, range_sample=range_sample_num, boundary="pad"
         )
         .min(skipna=True)
     )
-    ds_out.coords["range_sample"] = (
-        "range_sample",
-        np.arange(ds_out["range_sample"].size),
-    )  # reset range_sample to start from 0
-
-    # Attach attributes
-    ds_out.attrs = {
-        "binning_mode": "index",
-        "range_sample_num": range_sample_num,
-        "ping_num": ping_num,
-    }
+    _set_MVVBS_attrs(ds_MVBS)
+    ds_MVBS["Sv"] = ds_MVBS["Sv"].assign_attrs(
+        {
+            "comment": "MVBS binned on the basis of range_sample and ping number specified as index numbers",
```

Comment:
@leewujung pay attention to this addition. I couldn't come up with a `cell_methods` alternative. I was getting hung up with the specification of bins based on index numbers rather than physical units.
FYI, `comment` is a CF attribute.

Reply:
Based on what you had for the case when MVBS is computed based on physical units (actual range in meters and ping time), how about something like below?
```
{
            "cell_methods": (
                f"ping_time: mean (interval: {ping_num} pings "
                "comment: ping_time is the interval start) "
                f"echo_range: mean (interval: {range_sample_num} samples along range "
                "comment: echo_range is the interval start)"
            ),
            "binning_mode": "sample number",
            "range_meter_interval": str(range_sample_num) + "samples along range",
            "ping_time_interval": str(ping_num) + "pings",
}
```

Reply:
Adding link to where I discussed CF `cell_methods`: https://github.com/OSOceanAcoustics/echopype/pull/594#pullrequestreview-921175073

Reply:
A challenge with your suggestion is the interval units string. [Per CF](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#recording-spacing-original-data): 
> The syntax is (`interval`: *value unit*), where *value* is a numerical value and *unit* is a string that can be recognized by UNIDATA’s Udunits package [[UDUNITS]](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#UDUNITS).

So, strictly speaking, using "pings" and "samples along range" would not be compliant. The difficulty is in trying to specify a constant interval that's based on the index number rather than the dimension units. Though CF also says this:

> The *unit* will usually be dimensionally equivalent to the unit of the corresponding dimension, but this is not required (which allows, for example, the interval for a standard deviation calculated from points evenly spaced in distance along a parallel to be reported in units of length even if the zonal coordinate of the cells is given in degrees).

Let me chew on this.

Reply:
```
"range_meter_interval": str(range_sample_num) + "samples along range",
"ping_time_interval": str(ping_num) + "pings",
```
This seems confusing, since the value after "range_meter_interval" will not be in meters, nor will the value after "ping_time_interval" be in time units. It seems like the goal of these custom echopype attributes was to record the function parameters used, as is. If we find a way to settle on using `cell_methods`, the linkage between `ping_time` and `echo_range` and the binning method (mean) & interval values used would be explicit, so I'd then vote for retaining the original attribute names of "range_sample_num" and "ping_num".

Reply:
I am not entirely understanding what you are proposing, but when proposing the `cell_methods` I actually forgot to check the attribute names. It should be
```
"range_sample_interval": str(range_sample_num) + "samples along range",
"ping_interval": str(ping_num) + "pings",
```
ie "range_**sample**_interval" instead of "range_meter_interval", "ping_~time~_interval" instead of "ping_time_interval"